src/expandableCalendar/index.tsx

```diff
@@ -255,29 +259,35 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
     }
   }, []);
 
-  useEffect(() => {
-    // date was changed from AgendaList, arrows or scroll
-    scrollToDate(date);
-  }, [date]);
-
   const handleScreenReaderStatus = (screenReaderEnabled: any) => {
     setScreenReaderEnabled(screenReaderEnabled);
   };
 
   /** Scroll */
+  const autoScroll = useRef(false);
+
+  const scrollToDate = throttle((date: string) => {
+    autoScroll.current = true;
 
-  const scrollToDate = (date: string) => {
     if (!horizontal) {
       calendarList?.current?.scrollToDay(date, 0, true);
     } else if (getYear(date) !== visibleYear.current || getMonth(date) !== visibleMonth.current) {
       // don't scroll if the month is already visible
       calendarList?.current?.scrollToMonth(date);
     }
-  };
+  }, 100, {leading: false, trailing: true});
```

Comment:
I think it's better to set leading to true and increase the throttle time. So the first press will take into account and quick presses after that will be throttled.

Reply:
Done